src/signals/incident/components/IncidentForm/index.test.js

```diff
@@ -474,5 +474,48 @@ describe('<IncidentForm />', () => {
 
       expect(screen.queryByRole('alert')).not.toBeInTheDocument()
     })
+
+    it('should trigger isBlocking validation', async () => {
+      const props = {
+        ...defaultProps,
+        fieldConfig: {
+          controls: {
+            error: {
+              meta: {},
+              render: FormComponents.GlobalError,
+            },
+            $field_0: {
+              isStatic: false,
+              render: IncidentNavigation,
+            },
+            phone: {
+              meta: {
+                label: 'Blocking answer test',
+                isVisible: true,
+                type: 'alert',
+              },
+              render: FormComponents.PlainText,
+              options: {
+                validators: ['isBlockingAnswer'],
```

Comment:
maybe convert this to isBlocking? cause its normally based on an answer but can be used without technically

Reply:
If we agree, then we pass it to PO. needs to be communicated to other users of signals.